ui/js/graphs.js

```diff
@@ -506,12 +527,14 @@ perf.controller('GraphsCtrl', [
                            'get_signature_properties/?signatures=' +
                            partialSeries.signature).then(function(response) {
                              var data = response.data;
+                             var subtest_signatures = data[0].subtest_signatures;
                              if (!propsHash[partialSeries.project]) {
                                propsHash[partialSeries.project] = {};
                              }
                              propsHash[partialSeries.project][partialSeries.signature] = data[0];
+                             return subtest_signatures;
```

Comment:
Lines 530, 535 aren't necessary. We already set the subtestSignatures property in PhSeries: https://github.com/mozilla/treeherder/blob/master/ui/js/perf.js#L17